homeassistant/components/homematic/entity.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, config):
 
     async def async_added_to_hass(self):
         """Load data init callbacks."""
-        await self.hass.async_add_job(self._subscribe_homematic_events)
```

Comment:
The only danger is if one day it will contain I/O.

Reply:
I could switch to to `async_add_executor_job` instead if @danielperna84 thinks there might be some refactoring coming in the future that will do I/O??

Reply:
I'm not that confident when it comes to async, so I can't judge the result of this PR in general. But to answer the question: no, it's not planned to introduce IO in that spot.

Reply:
Thanks